packages/devextreme/js/ui/context_menu/ui.menu_base.js

```diff
@@ -214,7 +214,7 @@ class MenuBase extends HierarchicalCollectionWidget {
         const { text } = itemData;
         const $itemContainer = $('<span>').addClass(DX_MENU_ITEM_CAPTION_CLASS);
```

Comment:
we create itemContainer and define itemText, but in case text is undefined we dont need it right?
lest return earlier in this case, like
```suggestion
        if(!text) return undefined;
```